### ISSUE TITLE
V5.0.0-b14

### ISSUE THREAD

**Issue by NicoMarkus** [fondberg/spotcast#500]
### The problem

When I want to install the integration, I get straight away "Open website
This step requires you to visit an external website to be completed." and get redirected to the external website. I'm not being asked for credentials or tokens.

### System Health details

System health ok 

### What type of installation are you running?

Home Assistant Container

### Diagnostics information

Test

**Comment by fcusson** [comment]
@NicoMarkus, we're you asked for a client ID and client secret first?

If not, did you provide them in the past and are trying to replace them?

Do you have any error logs? 

Being redirected to another site is normal that's a required step for the Oauth credentials, you should get redirected to spotify allow Home Assistant to get access to your information.

The process for the config are supposed to be 

1. Check if oauth app exist, if not ask for client ID and secret
2. Get access approval from Spotify
3. Ask for sp_dc and sp_key

**Comment by NicoMarkus** [comment]
Hello.  I did manage to setup this before. But I changed the account. I made the app in the developper portal. I removed spotcast and reinstalled it. It doesn't ask for client ID and secret or sp_dc and sp_key. 

When I add the intergration, nothing is asked, and I'm directed straight away to https://accounts.spotify.com/authorize?response_type=xxxxxx. This is where I get INVALID_CLIENT: Failed to get client. 

There are no errors in the logs.
